lib/ronad/maybe.rb

```diff
@@ -2,6 +2,12 @@
 
 module Ronad
   class Maybe < Monad
+    # Creates a single Maybe from multiple values, shorting on the first
+    # non-nil value.
+    def self.from_multiple_values(*vals)
+      new vals.detect { |v| Monad.new(v).value != nil }
```

Comment:
Ronad::Maybe#self.from_multiple_values has the variable name 'v'

[Read more about it here.](https://github.com/troessner/reek/blob/master/docs/Uncommunicative-Variable-Name.md)